scania-digital-design-system/tegel
# feat(docs) prefix info in migration docs

**Describe pull-request**  
A draft on how we could inform about the prefix change.

**Solving issue**  
Fixes: [DTS-1764](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?assignee=63ff6d6c314f5088138187f0&selectedIssue=DTS-1764)

**How to test**  
1. Go to Storybook
2. Check in "Migrating from components v4" document.  
3. Look at the newly added info in the document

[DTS-1764]: https://tegel.atlassian.net/browse/DTS-1764?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
